src/components/Game/Slots/Slots.scss

```diff
@@ -1,17 +1,28 @@
 @import "../../../styles/abstracts/variables";
 
+.SlotsSection {
+    display: grid;
+    grid-template-columns: repeat(7, 1fr);
+    grid-template-rows: 50px 2px;
+    grid-gap: 0 5px;
+}
+
 .Slots {
+    grid-column: 1 / -1;
     display: grid;
-    grid-gap: 5px;
     grid-template-columns: repeat(7, 1fr);
-    align-items: center;
+    grid-gap: 5px;
+    list-style: none;
 }
 
 .Slot {
-    height: 50px;
-    border-bottom: 2px solid $color-black;
     text-transform: uppercase;
     text-align: center;
     font-size: 40px;
     line-height: 50px;
 }
+
+.SlotBottom {
+    height: 2px;
+    background: $color-black;
+}
```

Comment:
it would be much better to not add HTML elements for styling purposes only, it's a bad practice to do so

I believe the same visual effect should be achievable with `border-bottom` or pseudoelement (:after), if a slot is empty, let's display it anyway with no text inside it, obviously this requires some changes in render function